aks/aks-post-processing.sh

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET --tenant $ARM_TENANT_ID
 
-export MCRGNAME=$(az aks show --resource-group $RGNAME --name $AKSNAME --query nodeResourceGroup -o json | jq -r '.')
+export MCRGNAME=$(az aks show --resource-group $RGNAME --name $AKSNAME --query nodeResourceGroup -o json --subscription $ARM_SUBSCRIPTION_ID | jq -r '.')
 
-export VMNODES=$(az vm list --resource-group $MCRGNAME -o json | jq -r '.[] | select (.tags.poolName | contains("'$NODEPOOLNAME'")) | .name')
+export VMNODES=$(az vm list --resource-group $MCRGNAME -o json --subscription $ARM_SUBSCRIPTION_ID | jq -r '.[] | select (.tags.poolName | contains("'$NODEPOOLNAME'")) | .name')
 
 for i in $VMNODES
  do
-   az vm run-command invoke -g $MCRGNAME -n $i --command-id RunShellScript --scripts \
+   az vm run-command invoke -g $MCRGNAME -n $i --subscription $ARM_SUBSCRIPTION_ID --command-id RunShellScript --scripts \
    "sudo sed -i -r 's|^(GRUB_CMDLINE_LINUX_DEFAULT=)\"(.*.)\"|\1\"\2 swapaccount=1\"|' \
    /etc/default/grub.d/50-cloudimg-settings.cfg && sudo update-grub"
-   az vm restart -g $MCRGNAME -n $i
+   az vm restart -g $MCRGNAME -n $i --subscription $ARM_SUBSCRIPTION_ID
 done
-az logout
\ No newline at end of file
+ az logout
```

Comment:
Actually, swap accounting is no longer required from 1.5.1 onwards (as long as the nodes themselves have swap disabled which is true on AKS now).

So, I'd suggest just removing post-processing.sh and removing the dependency on it from the `k8sresources.tf`.

Reply:
I will do that in my next PR with GKE :) after testing it on both GKE and AKS :)